subsys/bluetooth/shell/vcs_client.c

```diff
@@ -622,9 +622,9 @@ static int cmd_vcs_client_vocs_offset_set(const struct shell *sh,
 		return -ENOEXEC;
 	}
 
-	if (offset > UINT8_MAX || offset < -UINT8_MAX) {
+	if (offset > BT_VOCS_MAX_OFFSET || offset < -UINT8_MAX) {
```

Comment:
Why is only one of the limits replaced here?
Without looking at the code in detail, this case seems very analogous to the one in the other file, where both limits have been replaced.

Reply:
Good question; not sure how that happened. Must've forgot to save the file. Thanks!